app/src/main/java/edu/uc/hornunjb/weatherwear/dto/Weather.kt

```diff
@@ -40,48 +16,12 @@ class Weather {
     var icon: String? = null
 }
 
-class Clouds {
-    @SerializedName("all")
-    var all: Float = 0.toFloat()
-}
 
-class Rain {
-    @SerializedName("3h")
-    var h3: Float = 0.toFloat()
-}
 
-class Wind {
-    @SerializedName("speed")
-    var speed: Float = 0.toFloat()
-    @SerializedName("deg")
-    var deg: Float = 0.toFloat()
-}
 
-class Main {
-    @SerializedName("temp")
-    var temp: Float = 0.toFloat()
-    @SerializedName("humidity")
-    var humidity: Float = 0.toFloat()
-    @SerializedName("pressure")
-    var pressure: Float = 0.toFloat()
-    @SerializedName("temp_min")
-    var temp_min: Float = 0.toFloat()
-    @SerializedName("temp_max")
-    var temp_max: Float = 0.toFloat()
-}
 
-class Sys {
-    @SerializedName("country")
-    var country: String? = null
-    @SerializedName("sunrise")
-    var sunrise: Long = 0
-    @SerializedName("sunset")
-    var sunset: Long = 0
-}
 
-class Coord {
-    @SerializedName("lon")
-    var lon: Float = 0.toFloat()
-    @SerializedName("lat")
-    var lat: Float = 0.toFloat()
-}
```

Comment:
I thought it would be better practice to put this class in its own file. What I found was it is better for classes to be alone.